_layouts/home.html

```diff
@@ -58,8 +58,8 @@ <h3>Ole Schäfer</h3>
                     <h3>Simon Sprankel</h3>
                     <img src="/images/call/Simon.jpg" alt="Simon">
                     <ul class="list-group">
-                        <li class="list-group-item">Automatisierungen</li>
-                        <li class="list-group-item">Prozessoptimierungen</li>
+                        <li class="list-group-item">Codequalität</li>
```

Comment:
I'm not sure anyone wants to talk about this :D I would love to keep the previous ones, but remove this :thinking:

Reply:
@sprankhub , as a new developer I would love to hear some concepts regarding this topic. 
Just as a general entry point. And you're definitely crazy about that, so that should definitely be one of the points in my eyes.

Reply:
Yeah, but I guess our target group is merchants and not so much developers :thinking: And I think for those the old two points are much more relevant. But up to you, I pretty much talk about anything :stuck_out_tongue:

Reply:
Okay, I gave you both back, but removed deployments and Magento 1 instead.
That's a fair switch.